spec/features/public_articles_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+
+feature "Sprawdzanie opublikowanych artykulów" do
+   include_context "admin login"
+   before do 
+     
+     person1 = Person.create(name: 'Adam', surname: 'Kapusta', email: 'a@k.com', sex: 'mężczyzna', roles: ['autor'])
```

Comment:
We wszystkich wywołaniach `create` powinien być wykrzyknik - jeśli obiekt się nie utworzy, to zostanie rzucony wyjątek.